application/src/main/resources/ecc.yml

```diff
@@ -86,6 +86,15 @@ repair:
     time: 7
     unit: days
   ##
+  ##  The unit of time granularity for priority calculation, can be HOURS, MINUTES, or SECONDS.
+  ##  It determines how often the priority is recalculated.
```

Comment:
"It determines how often the priority is recalculated." this is not true. It's used to calculate priority not "how often".